packages/manager/apps/pci-instances/src/vite-hmr.ts

```diff
@@ -0,0 +1,5 @@
+if (import.meta.hot) {
+  import.meta.hot.on('iframe-reload', () => {
+    window.location.reload();
+  });
```

Comment:
Are we sure we want to keep that ?

Reply:
I'm not sure about that, I'll say no but all `pci-*` apps have the configuration file